docs/user_manual/projects.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: page
+title: "6. Projects"
+section: 6
+---
+
+
+A project contains information about a set of studies that may comprise many different samples, experiments and
+sequencer runs. Each study and sample in MISO belongs to a specific project. By association, a sample's libraries and
+dilutions belong to the same project. Projects can also have overviews, which hold information about a project
+proposal.
```

Comment:
`design and status` instead of `proposal`? (Since they're theoretically useful for tracking how closely the current state matches the project's design). Minor preference.